corehq/sql_db/routers.py

```diff
@@ -169,7 +169,8 @@ def get_db_for_partitioned_model(model, hints):
         return plproxy_config.proxy_db
     if HINT_USING in hints:
         db = hints[HINT_USING]
-        assert db in get_db_aliases_for_partitioned_query()
+        assert db in get_db_aliases_for_partitioned_query(), "{} not in {}".format(
+            db, ", ".join(get_db_aliases_for_partitioned_query()))
```

Comment:
oh! did we see an instance where this happened?
might want to fetch `get_db_aliases_for_partitioned_query` once and store it instead of calling it twice.

Reply:
It should be very cheap though this function does get called with a very high frequency.

Reply:
huh, thought I updated this to not call `get_db_aliases_for_partitioned_query ` twice, must have lost that commit. Going to leave this as-is, since the second call doesn't happen unless the assertion fails.

fwiw I came across this while working in the shell, not because of a code problem